plugins/outputs/zabbix/README.md

```diff
@@ -0,0 +1,406 @@
+  ## Address and (optional) port of the Zabbix server
+  address = "zabbix.example.com:10051"
+
+  ## Send metrics as type "Zabbix agent (active)"
+  # agent_active = false
+
+  ## Add prefix to all keys sent to Zabbix.
+  # prefix = "telegraf."
+
+  ## Name of the tag that contains the host name
+  # host_tag = "host"
+
+  ## Skip measurement prefix to all keys sent to Zabbix.
+  # skip_measurement_prefix = false
+
+  ## This field will be sent as HostMetadata to Zabbix Server to autoregister the host.
+  ## To enable this feature, this option must be set to a value other than "".
+  # autoregister = ""
+
+  ## Interval to resend auto-registration data to Zabbix.
+  ## Only applies if autoregister feature is enabled.
+  ## This value is a lower limit, the actual resend should be triggered by the next flush interval.
+  # autoregister_resend_interval = "30m"
+
+  ## Interval to send LLD data to Zabbix.
+  ## This value is a lower limit, the actual resend should be triggered by the next flush interval.
+  # lld_send_interval = "10m"
+
+  ## Interval to delete stored LLD known data and start capturing it again.
+  ## This value is a lower limit, the actual resend should be triggered by the next flush interval.
+  # lld_clear_interval = "1h"
+```
+
+### agent_active
+
+The `request` value in the package sent to Zabbix should be different if the
+items configured in Zabbix are [Zabbix trapper][zabbixtrapper] or
+[Zabbix agent (active)][zabbixagentactive].
+
+`agent_active = false` will send data as _sender data_, expecting trapper items.
+
+`agent_active = true` will send data as _agent data_, expecting active Zabbix
+agent items.
+
+[zabbixtrapper]: https://www.zabbix.com/documentation/6.4/en/manual/config/items/itemtypes/trapper?hl=Trapper
+[zabbixagentactive]: https://www.zabbix.com/documentation/6.4/en/manual/config/items/itemtypes/zabbix_agent
+
+### prefix
+
+We can set a prefix that should be added to all Zabbix keys.
+
+This is configurable with the option `prefix`, set by default to `telegraf.`.
+
+Example how the configuration `prefix = "telegraf."` will generate the Zabbix
+keys given a Telegraf metric:
+
+```diff
+- measurement,host=hostname valueA=0,valueB=1
++ telegraf.measurement.valueA
++ telegraf.measurement.valueB
+```
+
+### skip_measurement_prefix
+
+We can skip the measurement prefix added to all Zabbix keys.
+
+Example with `skip_measurement_prefix = true"` and `prefix = "telegraf."`:
+
+```diff
+- measurement,host=hostname valueA=0,valueB=1
++ telegraf.valueA
++ telegraf.valueB
+```
+
+Example with `skip_measurement_prefix = true"` and `prefix = ""`:
+
+```diff
+- measurement,host=hostname valueA=0,valueB=1
++ valueA
++ valueB
+```
+
+### autoregister
+
+If this field is active, Telegraf will send an
+[autoregister request][autoregisterrequest] to Zabbix, using the content of
+this field as the [HostMetadata][hostmetadata].
+
+One request is sent for each of the different values seen by Telegraf for the
+`host` tag.
+
```

Comment:
What if the host tag does not exist?

Reply:
https://github.com/influxdata/telegraf/pull/13739/files#diff-a148a78339729b36ca6334c99c1c6168e65a5b4369e6ed325a613dbbb9a9ddceR233-R240